v4/source/bots.yaml

```diff
@@ -240,3 +238,148 @@
           $ref: '#/responses/Unauthorized'
         '403':
           $ref: '#/responses/Forbidden'
+
+  /bots/{bot_user_id}/icon:
+    get:
+      tags:
+        - bots
+      summary: Get bot's LHS icon
+      description: |
+        Get a bot's LHS icon image based on bot_user_id string parameter.
+        ##### Permissions
+        Must be logged in.
```

Comment:
Should we add here that they also need to have permission to view that bot user? 🤔

Reply:
The only permission `get` requires is `view_members`. Which seems to be the default permission granted to all users. Didn't see other examples of it so i went with just the default description of `must be logged in`. Could change it if that's the new convention we want to use going forward.

Reply:
I think it's fine to adopt the default description, but I suspect all of this needs to be updated to take into context guest accounts which explicitly don't have the usual permissions. cc @jespino

Reply:
Yes, exactly the `view_members` is granted to all users, but when guest accounts arrive, the guest users wont have that permission at all, so it only will be able to see users with direct relation (being in the same channel or being in a direct/group conversation).